persist/file-adapter/adapter.go

```diff
@@ -113,11 +113,29 @@ func (a *Adapter) AddPolicy(sec string, ptype string, rule []string) error {
 	return errors.New("not implemented")
 }
 
+// AddPolicies adds policy rules to the storage.
+func (a *Adapter) AddPolicies(secs []string, ptypes []string, rules [][]string) []error {
+	errs := make([]error, 0)
+	for i := 0 ; i < len(secs) ; i++ {
+		errs = append(errs, errors.New("not implemented"))
+	}
+	return errs
+}
+
 // RemovePolicy removes a policy rule from the storage.
 func (a *Adapter) RemovePolicy(sec string, ptype string, rule []string) error {
 	return errors.New("not implemented")
 }
 
+// RemovePolicies removes policy rules from the storage.
+func (a *Adapter) RemovePolicies(secs []string, ptypes []string, rules [][]string) []error {
```

Comment:
I feel better, something like: `func (a *Adapter) RemovePolicies(secs []string, ptypes []string, rules [][]string) []error`.

Reply:
Agree with @nodece , we only have two ptypes: p and g. The user call it at most twice. This is OK.

Reply:
Same reason as above, we want ```[]string```.